desc/objectives/objective_funs.py

```diff
@@ -885,6 +933,7 @@ def jit(self):
         except AttributeError:
             pass
         self.compute_scaled = jit(self.compute_scaled)
+        self.compute_scaled2 = self.compute_scaled
```

Comment:
is this used?

Reply:
no it was a debug thing, thanks for catching